docs/src/introduction.md

```diff
@@ -58,6 +59,32 @@ p4est_connectivity_destroy(connectivity)
 ```
 
 
+## [`PointerWrapper`s](@id pointer_wrappers)
+
+As we have seen in the [previous section](@ref translation_guidelines) many functions
+provided by [`p4est`](https://github.com/cburstedde/p4est) return pointers to `struct`s,
+which requires to [`unsafe_load`](https://docs.julialang.org/en/v1/base/c/#Base.unsafe_load)
+the pointer in order to access the underlying data.
+[P4est.jl](https://github.com/trixi-framework/P4est.jl) offers a convenient way to avoid
+using `unsafe_load` whenever it is necessary by using a [`PointerWrapper`](@ref).
+If you, e.g., have a pointer to a `p4est_connectivity` (i.e. an object of type `Ptr{p4est_connectivity}`)
+called `connectivity`, you can use `connectivity_pw = PointerWrapper(connectivity)` to obtain
+a wrapped version of the pointer, where the underlying data can be accessed simply by
+`connectivity_pw.num_trees[]` without the need to call `unsafe_load` manually. This works even for nested
+structures, where the data of a `struct` is, again, a `struct`. The example from above, but now
```

Comment:
"example from above" is ambiguous; maybe this can be made more precise or with a link to it?

Reply:
I can give a link to the section "Translation guidelines", where the example is given, but can I also give a link to the example itself?

Reply:
Whatever you think works best. If a direct link to the example is possible, then why not?

Reply:
A modified and extended the example in [93fa7ea](https://github.com/trixi-framework/P4est.jl/pull/67/commits/93fa7ea2a0f2f7b8093a4fbae6798e3360e62b09). So it is not really the same example as above. Therefore, I also don't refer to it anymore.